src/beerapi.py

```diff
@@ -31,14 +32,16 @@ def get_inventory():
     Returns: The inventory or an emtpy string if the communication failed.
```

Comment:
Looks Good. But for your get_inventory() function documentation I suggest getting rid of the extra line spacing to make everything as tight as possible to save space.